tests/specification/deepCopy.js

```diff
@@ -1,9 +1,9 @@
+/* eslint-disable comma-dangle */
```

Comment:
please do not not change the code style in a pull request about tests. If you want to change the styles open an issue, discuss about it and then maybe make a separate PR for it

Reply:
okay. Noted. I think my default formatter changed it though.